src/index.js

```diff
@@ -43,6 +45,14 @@ class NoSleep {
       this._addSourceToVideo(this.noSleepVideo, "webm", webm);
       this._addSourceToVideo(this.noSleepVideo, "mp4", mp4);
 
+      // For iOS >15 video needs to be on the document to work as a wake lock
+      Object.assign(this.noSleepVideo.style, {
+        position: "absolute",
+        left: "-100%",
+        top: "-100%",
```

Comment:
    "position: absolute; top: -100%" alone should suffice.
    Demo (bookmarklet): `javascript: void document.getElementsByTagName( "body" )[0].appendChild( document.createRange().createContextualFragment( "<div style='position: absolute; top: -100%; background-color: #F00'>XXXXXXXX</div>" ) );`